src/Components/Artwork/Details/PartnerOfferedPrice.tsx

```diff
@@ -0,0 +1,80 @@
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Flex, Text } from "@artsy/palette"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { SaleMessage } from "./Details"
+import { PartnerOfferedPriceQuery } from "__generated__/PartnerOfferedPriceQuery.graphql"
+import { PartnerOfferedPrice_artwork$data } from "__generated__/PartnerOfferedPrice_artwork.graphql"
+import { Details_artwork$data } from "__generated__/Details_artwork.graphql"
+
+interface PartnerOfferedPriceProps {
+  artwork: PartnerOfferedPrice_artwork$data
+  fallback: object
+}
+
+const PartnerOfferedPrice: React.FC<PartnerOfferedPriceProps> = ({
+  artwork,
+  fallback,
+}) => {
+  const partnerOffer = artwork?.collectorSignals?.partnerOffer
+  const offeredPrice = partnerOffer?.priceWithDiscount?.display
+
+  return partnerOffer ? <Text lineHeight="22px">{offeredPrice}</Text> : fallback
+}
+
+const PartnerOfferedPriceFragmentContainer = createFragmentContainer(
+  PartnerOfferedPrice,
+  {
+    artwork: graphql`
+      fragment PartnerOfferedPrice_artwork on Artwork {
+        collectorSignals {
+          partnerOffer {
+            endAt
+            priceWithDiscount {
+              display
+            }
+          }
+        }
+      }
+    `,
+  }
+)
+
+interface PartnerOfferedPriceQueryRendererProps {
+  id: string
+  artwork: Details_artwork$data
+}
+
+export const PartnerOfferedPriceQueryRenderer: FC<PartnerOfferedPriceQueryRendererProps> = ({
+  id,
+  artwork,
+}) => {
+  return (
+    <SystemQueryRenderer<PartnerOfferedPriceQuery>
+      lazyLoad
+      query={graphql`
+        query PartnerOfferedPriceQuery($id: String!) {
+          artwork(id: $id) {
+            ...PartnerOfferedPrice_artwork
+          }
+        }
+      `}
+      placeholder={<SaleMessage artwork={artwork} />}
+      variables={{ id }}
+      render={({ error, props }) => {
+        const publicPrice = <SaleMessage artwork={artwork} />
+
+        if (error || !props?.artwork) {
+          return publicPrice
+        }
+
+        return (
+          <PartnerOfferedPriceFragmentContainer
```

Comment:
Can maybe include a subtle/nice little 'pulse' animation like in https://github.com/artsy/force/blob/4d6f44a3f8cf18281e780a7c2f26b3cebbfc08f7/src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx#L20